Tools/2MGFX/Program.cs

```diff
@@ -7,6 +7,12 @@ public static class Program
     {
         public static int Main(string[] args)
         {
+            if (!Environment.Is64BitOperatingSystem)
```

Comment:
There are several ways to force a AnyCPU assembly to run as 32bit on a 64bit OS.  This error message would be skipped in those cases.  What we actually want to check is if the process is 64bit or not.